### ISSUE TITLE
Default font size and density too thin on VsCode Linux

### ISSUE THREAD

**Issue by mubaidr** [githubnext/monaspace#139]
Problem: Font weight for default is too thin. Using same VsCode settings for different OS (Windows 10 in my case works fine, font is rendered perfectly)

Screenshot of Issue: 
![Screenshot_20231119_230651](https://github.com/githubnext/monaspace/assets/2222702/947fec05-aa88-40f2-ab51-c1470baaa59e)

VsCode settings: 

```json
"editor.fontFamily": "'Monaspace Neon Var'",
  "editor.fontLigatures": "'calt', 'liga', 'dlig', 'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'ss07', 'ss08', 'ss09', 'ss10'",
  "editor.fontSize": 16,
  "editor.fontWeight": "400",
```

OS: 
```bash
Operating System: Manjaro Linux 
KDE Plasma Version: 5.27.9
KDE Frameworks Version: 5.112.0
Qt Version: 5.15.11
Kernel Version: 6.6.1-1-MANJARO (64-bit)
Graphics Platform: X11
Processors: 8 × Intel® Core™ i7-8550U CPU @ 1.80GHz
Memory: 7.7 GiB of RAM
Graphics Processor: Mesa Intel® UHD Graphics 620
Manufacturer: Dell Inc.
Product Name: Inspiron 5570
```

VsCode: 
```bash
Version: 1.85.0-insider
Commit: 54b8f6a80aaad7489b75e9293b8d7a1700bed815
Date: 2023-11-17T10:49:23.290Z (2 days ago)
Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Code-Insiders/1.85.0-insider Chrome/114.0.5735.289 Electron/25.9.4 Safari/537.36
```

**Comment by mubaidr** [comment]
Update: 

- linux does not recognize this font as `monospace` font. 
- this same issue persists in console application too (konsole in my case)

![Screenshot_20231205_133203](https://github.com/githubnext/monaspace/assets/2222702/29e06490-ae82-4c2c-a065-24840e57003a)

**Comment by TZECHIN6** [comment]
https://github.com/githubnext/monaspace/issues/182

Mine is just too thick, look like bold

**Comment by heathercran** [comment]
This is because you're using the variable fonts, which VS Code doesn't fully support yet. If you install the static versions of the fonts instead, it should look as expected!

**Comment by mubaidr** [comment]
> Update:
> 
> * linux does not recognize this font as `monospace` font.
> * this same issue persists in console application too (konsole in my case)
> 
> ![Screenshot_20231205_133203](https://private-user-images.githubusercontent.com/2222702/287974509-29e06490-ae82-4c2c-a065-24840e57003a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYzNjAwMTksIm5iZiI6MTcxNjM1OTcxOSwicGF0aCI6Ii8yMjIyNzAyLzI4Nzk3NDUwOS0yOWUwNjQ5MC1hZTgyLTRjMmMtYTA2NS0yNDg0MGU1NzAwM2EucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjJUMDYzNTE5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDMyY2NiNzYwZTAyMzc3ZDZmNmNjMmE1YzBmN2JmZThlOWQyNzI0YmQ0NzAwN2QxMDljMDY1NTYyZDAzMDU5YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lLae-tHdr26abm52oXaE_x72iZxs-KTcwIsILiU_0B4)

Any update on this?

**Comment by heathercran** [comment]
> > Update:
> > 
> > * linux does not recognize this font as `monospace` font.
> > * this same issue persists in console application too (konsole in my case)
> > 
> > ![Screenshot_20231205_133203](https://private-user-images.githubusercontent.com/2222702/287974509-29e06490-ae82-4c2c-a065-24840e57003a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYzNjAwMTksIm5iZiI6MTcxNjM1OTcxOSwicGF0aCI6Ii8yMjIyNzAyLzI4Nzk3NDUwOS0yOWUwNjQ5MC1hZTgyLTRjMmMtYTA2NS0yNDg0MGU1NzAwM2EucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjJUMDYzNTE5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDMyY2NiNzYwZTAyMzc3ZDZmNmNjMmE1YzBmN2JmZThlOWQyNzI0YmQ0NzAwN2QxMDljMDY1NTYyZDAzMDU5YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lLae-tHdr26abm52oXaE_x72iZxs-KTcwIsILiU_0B4)
> 
> Any update on this?

Yes! That issue was related to panose flags, which was fixed in version 1.1.
